packages/0x.js/package.json

```diff
@@ -15,7 +15,6 @@
     "main": "lib/src/index.js",
     "types": "lib/src/index.d.ts",
     "scripts": {
-        "watch_without_deps": "tsc -w",
```

Comment:
We don't need `watch_without_deps` any more. Running `yarn watch:ts` in the monorepo root will automatically watch all packages and their dependencies and only recompile them as needed. It's much faster than the old approach.